doc/cli_vs_yum.rst

```diff
@@ -49,6 +49,7 @@ Invoking ``dnf update`` or ``dnf upgrade``, in all their forms, has the same
 effect in DNF, with the latter being preferred. In YUM ``yum upgrade`` was
 exactly like ``yum --obsoletes update``.
 
+
```

Comment:
Thanks, can you squash the commits into one (we normally update the commits instead of adding new ones during the review process)?

After you do that, you'll notice the unrelated empty line above was added while moving the text around, can you remove it?

Otherwise LGTM.